glTFLoader/glTFLoader.csproj

```diff
@@ -1,101 +1,24 @@
-    <Compile Include="Schema\AnimationChannelTarget.cs" />
-    <Compile Include="Schema\AnimationSampler.cs" />
-    <Compile Include="Schema\Asset.cs" />
-    <Compile Include="Schema\Buffer.cs" />
-    <Compile Include="Schema\BufferView.cs" />
-    <Compile Include="Schema\Camera.cs" />
-    <Compile Include="Schema\CameraOrthographic.cs" />
-    <Compile Include="Schema\CameraPerspective.cs" />
-    <Compile Include="Schema\Extension.cs" />
-    <Compile Include="Schema\Extras.cs" />
-    <Compile Include="Schema\Gltf.cs" />
-    <Compile Include="Schema\GltfChildOfRootProperty.cs" />
-    <Compile Include="Schema\GltfProperty.cs" />
-    <Compile Include="Schema\Image.cs" />
-    <Compile Include="Schema\Material.cs" />
-    <Compile Include="Schema\MaterialNormalTextureInfo.cs" />
-    <Compile Include="Schema\MaterialOcclusionTextureInfo.cs" />
-    <Compile Include="Schema\MaterialPbrMetallicRoughness.cs" />
-    <Compile Include="Schema\Mesh.cs" />
-    <Compile Include="Schema\MeshPrimitive.cs" />
-    <Compile Include="Schema\Node.cs" />
-    <Compile Include="Schema\Sampler.cs" />
-    <Compile Include="Schema\Scene.cs" />
-    <Compile Include="Schema\Skin.cs" />
-    <Compile Include="Schema\Texture.cs" />
-    <Compile Include="Schema\TextureInfo.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="Package.nuspec">
-      <SubType>Designer</SubType>
-    </None>
-    <None Include="packages.config" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="..\Generator\Generator.csproj">
-      <Project>{ddd005c7-d63d-4707-9464-1bcb9fd00454}</Project>
-      <Name>Generator</Name>
-      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
-      <Private>False</Private>
-    </ProjectReference>
-    <ProjectReference Include="..\glTFLoader_Shared\glTFLoader_Shared.csproj">
-      <Project>{976ef0aa-4835-4b60-88b8-bf2fa02b571c}</Project>
-      <Name>glTFLoader_Shared</Name>
-    </ProjectReference>
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.3;net452</TargetFrameworks>
+    <Authors>Matthew and Xueru McMullan</Authors>
+    <Copyright>Copyright ©  2015</Copyright>
+    <Description>C# reference loader for glTF</Description>
+    <Company>Khronos Group</Company>
+    <Product>glTFLoader</Product>
+    <PackageId>glTFLoader</PackageId>
+    <GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+    <PackageProjectUrl>https://github.com/KhronosGroup/glTF/blob/master/loaders/CSharp/README.md</PackageProjectUrl>
+    <PackageLicenseUrl>https://github.com/KhronosGroup/glTF/blob/master/loaders/CSharp/LICENSE.md</PackageLicenseUrl>
+    <RepositoryUrl>https://github.com/KhronosGroup/glTF-CSharp-Loader</RepositoryUrl>
+    <PackageIconUrl>https://github.com/KhronosGroup/glTF/blob/master/specification/figures/gltf.png</PackageIconUrl>
+    <PackageTags>glTF loader openGL C#</PackageTags>

```

Comment:
We should probably remove openGL tag since glTF is no longer tied to any specific GL library.